### ISSUE TITLE
Improve the experience for first time arrivals

### ISSUE THREAD

**Issue by ItalyToast** [HHSmithy/PokerHandHistoryParser#51]
Currently the wiki is very limited and don't describe new features such as HandValidator or FileIdentifier. We don't have a good starting example atm.

**Comment by lukebelbina** [comment]
@ItalyToast Agree. I'll carve some time to think about the project/code-base and how to move forward. I need to chat w/ @BewareOfWombat also to see what version we are using at Smithy
